src/main/java/com/cs525/reversi/models/Game.java

```diff
@@ -1,48 +1,70 @@
 package com.cs525.reversi.models;
 
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.UpdateTimestamp;
+
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.NonNull;
-import org.hibernate.annotations.Type;
-
-import javax.persistence.*;
-import java.util.Date;
-import java.util.UUID;
 
 @Entity
 @Data
 @NoArgsConstructor
 public class Game {
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private int id;
-
-    @Type(type = "uuid-char")
-    @Column(unique = true, nullable = false)
-    @NonNull
-    private UUID uuid;
-
-    @ManyToOne(fetch = FetchType.EAGER, optional = false)
-    @NonNull
-    private User player1;
-
-    @ManyToOne(fetch = FetchType.EAGER, optional = false)
-    @NonNull
-    private User player2;
-
-    @NonNull
-    @Column(nullable = false)
-    private Date createdAt;
-
-    @NonNull
-    @Column(nullable = false)
-    private Date updatedAt;
-
-    @Enumerated(EnumType.STRING)
-    @NonNull
-    @Column(nullable = false)
-    private GameStatus status;
-
-    // nullable
-    private Player winner;
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int id;
+
+	@Type(type = "uuid-char")
+	@Column(nullable = false)
+	@NonNull
+	private UUID uuid;
+
+	@ManyToOne(fetch = FetchType.EAGER, optional = false)
+	@NonNull
+	private User player1;
+
+	@ManyToOne(fetch = FetchType.EAGER, optional = false)
+	@NonNull
+	private User player2;
+
+	@NonNull
+	@Temporal(TemporalType.TIMESTAMP)
+	@CreationTimestamp
+	private Date createdAt;
+
+	@NonNull
+	@Temporal(TemporalType.TIMESTAMP)
+	@UpdateTimestamp
+	private Date updatedAt;
```

Comment:
@MustafaMosad `updatedAt` needs to be updated when a cell has been updated... I don't think the `@UpdateTimestamp` will catch that... it would only catch if entire rows are added or removed